### PR TITLE
Admin can see skills for auctions vendor has bid on

### PR DESCRIPTION
**Pull request by jessieay**
* Replace start/end date with "status" to make more room on page
* Closes https://github.com/18F/micropurchase/issues/846

screen shot 2016-07-12 at 1 27 51 pm

### CODE REVIEW

File: app/views/admin/users/_bid_history.html.erb [hunk 1]
```diff
@@ -4,8 +4,8 @@
   <thead>
     <tr>
       <th scope="col">Auction</th>
-      <th scope="col">Start Date</th>
-      <th scope="col">End Date</th>
+      <th scope="col">Status</th>
+      <th scope="col">Skills</th>
```
Comment (harrisj) [inline app/views/admin/users/_bid_history.html.erb]: Skills seems good, but do we not need the auction end date anymore?

File: spec/view_models/admin/user_auction_view_model_spec.rb [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+  describe '#skills' do
     it 'should return a mm/dd/yy date in EST' do
-      auction = create(:auction, ended_at: '2016-04-01 00:00 UTC')
-      expect(Admin::UserAuctionViewModel.new(auction, user).end_date).to eq('03/31/2016 EDT')
+      auction = create(:auction)
+      skills = [create(:skill, name: 'sewing'), create(:skill, name: 'eating')]
+      auction.skills << skills
+
+      expect(Admin::UserAuctionViewModel.new(auction, user).skills).to eq(
+        'sewing, eating'
```
Comment (harrisj) [inline spec/view_models/admin/user_auction_view_model_spec.rb]: Should skills be sorted by name or such?

File: app/views/admin/users/_bid_history.html.erb (hunk 1 above)
Comment (jessieay) [inline app/views/admin/users/_bid_history.html.erb]: spoke with michael and he seemed to think it was Ok to replace start/end date with status in order to preserve minimal table width

File: spec/view_models/admin/user_auction_view_model_spec.rb (hunk 2 above)
Comment (jessieay) [inline spec/view_models/admin/user_auction_view_model_spec.rb]: yes! can use method I added to other PR for this :)

File: app/views/admin/users/_bid_history.html.erb (hunk 1 above)
Comment (harrisj) [inline app/views/admin/users/_bid_history.html.erb]: Okay. It looks like we sort the auctions in descending chronological order so I guess that'll be okay

File: spec/view_models/admin/user_auction_view_model_spec.rb (hunk 2 above)
Comment (harrisj) [inline spec/view_models/admin/user_auction_view_model_spec.rb]: I just merged that PR, so feel free to rebase

**Comment by jessieay** [comment]
@harrisj pushed updates! e4fa1385a74e59e8a89d6dff0a2e77887face8a2

**Comment by harrisj** [comment]
Everything is awesome! Everything is cool when you're part of a team! Merging...

**Pull request merged**